workaround CMake Error at doc/cmake_install.cmake:56 (file):
file cannot create directory: /pictures.  Maybe need administrative
privileges.

--- CMakeLists.txt.orig	2019-10-30 16:31:12.000000000 +0100
+++ CMakeLists.txt	2019-11-12 12:18:48.305932925 +0100
@@ -68,7 +68,6 @@
 ADD_SUBDIRECTORY(include)
 ADD_SUBDIRECTORY(src)
 ADD_SUBDIRECTORY(unitTesting)
-ADD_SUBDIRECTORY(doc)
 
 # **********************************
 # Robot_utils_sot_py PYTHON module *
